src/index.html

```diff
@@ -52,6 +55,37 @@ <h1 class="profile__name">Jacques Cousteau</h1>
         <p class="footer__copyright">&copy; 2023 Around The U.S.</p>
       </footer>
     </div>
+
+    <!-- Edit Avatar Modal -->
+    <div class="modal" id="edit-avatar-modal">
+      <div class="modal__container modal__container_avatar">
+        <button
+          class="modal__close"
+          id="close-avatar-button"
+          type="button"
+        ></button>
+        <form
+          name="avatar-form"
+          class="modal__form"
+          name="avatar-form"
+          id="avatar-form"
+          novalidate
+        >
+          <p class="modal__heading">Change profile picture</p>
+          <input
+            name="avatar"
+            placeholder="Image Link"
+            id="avatar-link"
+            type="url"
+            class="modal__form-input modal__form-input_type_avatar"
```

Comment:
`modal__form-input`

* The avatar form is not validated.


The inputs in any form have class `modal__input` . Please, use the same in the avatar form to make validation work